richtextfx/src/main/java/org/fxmisc/richtext/skin/ParagraphBox.java

```diff
@@ -69,9 +70,10 @@ private CaretOffsetX(double value) {
     public void setIndex(int index) { this.index.setValue(index); }
     public int getIndex() { return index.getValue(); }
 
-    public ParagraphBox(Paragraph<S> par, BiConsumer<? super TextExt, S> applyStyle) {
+    public ParagraphBox(Paragraph<S, PS> par, BiConsumer<? super TextExt, S> applyStyle, PS initialParagraphStyle,  BiConsumer<TextFlow, PS> applyParagraphStyle) {
         this.getStyleClass().add("paragraph-box");
         this.text = new ParagraphText<>(par, applyStyle);
+        applyParagraphStyle.accept(this.text, initialParagraphStyle);
```

Comment:
The applied style should be `par.getParagraphStyle()` instead of `initialParagraphStyle`.